ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -222,23 +222,92 @@ export default {
   timelineReplace: [
     {
       'locale': 'de',
-      'missingTranslations': true,
       'replaceSync': {
         'Cloud Of Darkness': 'Wolke der Dunkelheit',
+        'Stygian Arbor': 'stygisch Dorn',
+      },
+      'replaceText': {
+        'Anti-air Particle Beam': 'Flugabwehr-Partikelstrahl',
+        'Bad Vibrations': 'Starke Vibration',
+        'Deluge of Darkness': 'Sintflut der Dunkelheit',
+        'Earth-shattering Particle Beam': 'Erderschütternder Partikelstrahl',
+        'Empty Plane': 'Düsterer Himmel',
```

Comment:
Also just to mention it: The ID 4FC6  (https://xivapi.com/action/20422) has for some reason 2 spaces instead of one `Düsterer  Himmel`. Since its just a text replacement it should be okay.